mhkit/dolfyn/io/nortek.py

```diff
@@ -514,13 +514,13 @@ def read_user_cfg(self):
         cfg_u["time_between_bursts"] = tmp[4]  # counts
         cfg_u["adv"]["n_pings_per_burst"] = tmp[5]
         cfg_u["awac"]["avg_interval"] = tmp[6]
-        cfg_u["usr"]["n_beams"] = tmp[7]
-        TimCtrlReg = _int2binarray(tmp[8], 16).astype(int)
+        cfg_u["usr"]["n_beams"] = int(tmp[7])
+        TimCtrlReg = _int2binarray(tmp[8], 16)
         # From the nortek system integrator manual
         # (note: bit numbering is zero-based)
         cfg_u["usr"]["profile_mode"] = ["single", "continuous"][TimCtrlReg[1]]
```

Comment:
Seems easier to leave the conversion to ``int`` with the definition of ``TimCtrlReg`` instead of duplicating it. Unless TimCtrlReg[1] is taking a special value, add the int conversion here:
```suggestion
        cfg_u["usr"]["profile_mode"] = ["single", "continuous"][int(TimCtrlReg[1])]

Reply:
I reviewed the preceding and following lines more and think my original suggestion is incorrect. Like ``burst_mode``, it looks like ``profile_mode`` should be a bool and not converted to an int.